### ISSUE TITLE
Full View support

### ISSUE THREAD

**Issue by axet** [yang991178/fluent-reader#403]
Hello!

Your app supports multiply way of viewing the articles: Card, List, Magazine, Compact but missing Full article view. Same as Magazine view, but loading article in full size.

Thanks!

**Comment by deathxxx123** [comment]
Yes! I need full screen mode for reading news... not only Phone mode view. I have big monitor and i cannot use it for full screen reading. Why is this?
